src/main/java/com/github/games647/flexiblelogin/config/TextConfiguration.java

```diff
@@ -0,0 +1,214 @@
+package com.github.games647.flexiblelogin.config;
+
+import com.gmail.frogocomics.flexiblelogin.LogUtils;
+
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.Texts;
+import org.spongepowered.api.text.format.TextColors;
+import org.spongepowered.api.text.format.TextStyles;
+import org.spongepowered.api.util.TextMessageException;
+
+import ninja.leaping.configurate.objectmapping.Setting;
+import ninja.leaping.configurate.objectmapping.serialize.ConfigSerializable;
+
+@ConfigSerializable
+public class TextConfiguration {
+
+    private Text parsingTextError = Texts.of(TextStyles.BOLD, TextColors.AQUA, "FlexibleLogin ", TextColors.GRAY, "\u2759 ", TextStyles.NONE, TextColors.DARK_RED, "The text configuration cannot be parsed. Please check the configuration and try again.");
```

Comment:
The plugin name shouldn't be constant. If we change the name in the main class, it should affect all other source files automatically.